docs/plugins.md

```diff
@@ -140,10 +141,13 @@ Note: The following plugins have been supplied by our community and may not have
 | [Supervisord](https://github.com/statianzo/dokku-supervisord)                                     | [statianzo][]         | Compatible with 0.2.0 |
 | [Logging Supervisord](https://github.com/sehrope/dokku-logging-supervisord)                       | [sehrope][]           | Works with dokku @ [c77cbf1][] - no 0.2.0 compatibility |
 | [Forego](https://github.com/iskandar/dokku-forego)                                                | [iskandar][]          | Compatible with 0.2.x |
+| [Monit](https://github.com/cjblomqvist/dokku-monit)<sup>1</sup>                                   | [cjblomqvist][]       |                       |
 
 [c77cbf1]: https://github.com/progrium/dokku/commit/c77cbf1d3ae07f0eafb85082ed7edcae9e836147
 [28de3ec]: https://github.com/progrium/dokku/commit/28de3ecaa3231a223f83fd8d03f373308673bc40
 
+<sup>1</sup> This plugin is far from complete and a mere proof-of-concept of how Monit can be used to externally monitor dokku apps. Read the [README](https://github.com/cjblomqvist/dokku-monit) for more details or contact the author.
```

Comment:
Can you remove this comment? I'd like to stay away from this sort of stuff as it tends to get out of date.